# This message contains the adx plan datatype.

geometry_msgs/Point position
geometry_msgs/Vector3 speed
